packages/cozy-client/src/StackLink.js

```diff
@@ -1,8 +1,26 @@
 import { MutationTypes } from './queries/dsl'
 import CozyLink from './CozyLink'
+import { CozyClientDocument } from './types'
 
 /**
+ * Returns full documents after a bulk update
  *
+ * @param  {{ ok: boolean, id: string, rev: string }[]} updateAllResponse - Response from bulk docs
```

Comment:
We can also receive: 
`error` and `reason` if the update of a particular doc has failed. 

https://docs.couchdb.org/en/stable/api/database/bulk-api.html?highlight=_bulk_docs#bulk-documents-transaction-semantics

Since it'll not throw an error we should store these informations in the response. 

But I don't know the right way to handle the fail? Is it on the app's land to map on the response to check if something has failed and do something? Do we provide feedback from cozyclient?

Reply:
I have modified the code to be closer to what happens in cozy-pouch-link.


```
    const badResults = updateAllResponse.filter(x => !x.ok)
    if (badResults.length > 0) {
      console.warn('Error while bulk saving, bad results', badResults)
      throw new Error('Error while bulk saving, see above for errors')
    }
```

I think the app should refetch the query in this case.

> Do we provide feedback from cozyclient?

 I attached the bad documents inside the error if the developer needs it.

Reply:
I added the error/reason to the TS definition, I think we can finish this discussion in https://github.com/cozy/cozy-client/pull/995#discussion_r681871162.